api/src/lib/helpers.ts

```diff
@@ -355,3 +355,33 @@ export const validateEmail = (email: string): boolean => {
 export const replaceHTMLLineBreaks = (html: string): string => {
     return html.replace(/\n|\r\n|\n\r|\r/g, '<br>');
 };
+
+// Check if two arrays are equal.
+export const compareArrays = <T>(a: Array<T>, b: Array<T>): boolean => {
```

Comment:
Is it worth having a unit test to confirm this works as expected?

Reply:
Yeah, good call. I'll write some.